### ISSUE TITLE
Screen Flickering

### ISSUE THREAD

**Issue by lazybuggy** [ManishaW/Reptile1400#5]
In the sewer/certain scenes have the screen flicker, or have ominous edges

**Comment by lazybuggy** [comment]
@ManishaW what do you think of this?

**Comment by ManishaW** [comment]
Yes, sewer definitely needs to bring in the creepy vibes. I was thinking we could do the flickering lights in the hallway scene maybe? then have a mission to fix the wire in fuse box or something like that

**Comment by lazybuggy** [comment]
https://www.youtube.com/watch?v=99q6snbOJzw
